provider/resources.go

```
@@ -3756,7 +3756,7 @@ compatibility shim in favor of the new "name" field.`)
 						Transform: tfbridge.TransformJSONDocument,
 					},
 				},
-				Docs: &tfbridge.DocInfo{Source: "../../../../docs/resource/aws_s3_bucket_legacy.md"},
+				Docs: &tfbridge.DocInfo{Markdown: maybeReadFile("docs/resource/aws_s3_bucket_legacy.md")},
```

Fix to the silently ignored source here.